Dockerfile

```diff
@@ -50,6 +52,7 @@ WORKDIR /peerplays-core
 # Compile Peerplays
 RUN \
     BOOST_ROOT=$HOME/boost_1_67_0 && \
+    export CC=gcc-5 ; export CXX=g++-5\
```

Comment:
These changes are not required. The build is compatible with gcc 7 too

Reply:
https://github.com/peerplays-network/peerplays/pull/212/commits/a07bcad7e80877b2a63fff73d1175c9f7ae477af